src/WorkOS.net/Services/_common/_interfaces/ListOptions.cs

```diff
@@ -24,5 +24,11 @@ public class ListOptions : BaseOptions
         /// </summary>
         [JsonProperty("limit")]
         public long? Limit { get; set; }
+
+        /// <summary>
+        /// Order of records returned by ascending or descending order.
+        /// </summary>
+        [JsonProperty("order")]
+        public PaginationOrder? Order { get; set; }
```

Comment:
similar to typescript, the question mark here marks it as optional right?

Reply:
Correct!